packages/web-frontend/src/components/Marker/markerIcons.js

```diff
@@ -17,11 +23,12 @@ import Help from '@material-ui/icons/Help';
 import CheckBox from '@material-ui/icons/CheckBox';
 
 // from https://thenounproject.com/ochavisual/collection/ocha-humanitarian-icons/
-import Cyclone from '../../images/cyclone.svg';
-import Earthquake from '../../images/earthquake.svg';
-import Tsunami from '../../images/tsunami.svg';
-import Volcano from '../../images/volcano.svg';
-import Flood from '../../images/flood.svg';
+
+import Cyclone from '!!react-svg-loader!../../images/cyclone.svg';
```

Comment:
Hey @chris-bes, I just saw this conversation. I have another PR open to move all the marker code to ui-components along with all the map-components from web-frontend. I ran into a similar issue there and resolved it in the rollup build config. So let's either wait until those changes are merged in to dev or come up with a quick work around so that you can move on. I'll send you a PM.

Reply:
@edmofro What do we think about hardcoding the disaster icons to tupaia orange as a quick fix for this temporary code? (As far as I can tell they are always displayed orange). Then when my map refactoring is complete this code will be deleted.

Reply:
Eeek yes sorry @tcaiger could you take this over now @chris-bes is off on Tamanu? Hardcoding to tupaia orange is totally fine with me.

Reply:
@edmofro - yes can do. My understanding is that this file is the last outstanding issue. Is that correct?

Reply:
Yes plus a merge from dev with some conflicts to resolve, and my accidental commit before lunch (was meant for a different branch). Thanks heaps for taking care of the svg stuff, I'll rebase the branch now to get rid of that commit and resolve the conflicts!

Reply:
Rebased and resolved conflicts. Hopefully I got it all right. Ready for your input on the svgs @tcaiger!